packages/default/scss/orgchart/_layout.scss

```diff
@@ -0,0 +1,92 @@
+@include exports("orgchart/layout") {
+
+    // OrgChart
+    .k-orgchart {
+        width: 100%;
+        box-sizing: border-box;
+        font-size: $orgchart-font-size;
+        font-family: $orgchart-font-family;
+        line-height: $orgchart-line-height;
+        display: flex;
+        justify-content: center;
+        overflow: auto;
+    }
+
+    .k-orgchart-container {
+        position: relative;
+    }
+
+    // Nodes
+    .k-orgchart-group {
+        gap: $orgchart-group-gap;
+    }
+
+    .k-orgchart-group.k-vstack > .k-orgchart-node-container {
+        gap: 0;
+    }
+
+    .k-orgchart-node-container {
+        gap: $orgchart-node-gap;
+    }
+
+    .k-orgchart-node-group-container {
+        @include border-radius( $orgchart-node-group-border-radius );
+        padding: $orgchart-node-group-padding-y $orgchart-node-group-padding-x;
+        border-width: $orgchart-node-group-border-width;
+        border-style: solid;
+    }
+
+    .k-orgchart-node-group-title {
+        margin: 0 0 $orgchart-node-group-title-margin-bottom;
+        font-size: $orgchart-node-group-title-font-size;
+        line-height: $orgchart-node-group-title-line-height;
+    }
+
+    .k-orgchart-node-group-subtitle {
+        margin: 0 0 $orgchart-node-group-subtitle-margin-bottom;
+        font-size: $orgchart-node-group-subtitle-font-size;
+    }
+
+    // Card
+    .k-orgchart-card {
+        width: $orgchart-card-width;
+        border-width: $orgchart-card-border-width;
+
+        .k-card-title {
+            margin-bottom: $orgchart-card-title-margin-bottom;
+            font-size: $orgchart-card-title-font-size;
+        }
+
+        .k-card-subtitle {
+            margin-bottom: $orgchart-card-subtitle-margin-bottom;
+            font-size: $orgchart-card-subtitle-font-size;
+        }
+
+        .k-card-body {
+            border-width: $orgchart-card-body-border-width;
+            border-style: solid;
+            flex-grow: 0;
+        }
+        .k-card-body .k-vbox {
+            margin: 0 $orgchart-card-body-vbox-margin-right 0 0;
+            min-width: 0;
+            min-height: $orgchart-card-body-vbox-min-height;
```

Comment:
not a good idea. some one might stack several `k-box` ... btw, use `k-vstack`, like you do above!